giraph-core/src/main/java/org/apache/giraph/graph/RetryableJobProgressTrackerClient.java

```diff
@@ -57,6 +68,10 @@
   private ThriftClientManager clientManager;
   /** Job progress tracker */
   private JobProgressTracker jobProgressTracker;
+  /** Cached value for number of retries */
+  private int numRetries = 1;
+  /** Cached value for wait time between retries */
+  private int retryWaitMsec;
```

Comment:
Do we need to initialize this to 1000?

Reply:
We don't need to. It will get initialized in the constructor. For consistency in the style, I will remove the initialization of numRertries too.